packages/react-core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -127,7 +127,7 @@ const DropdownBase: React.FunctionComponent<DropdownProps> = ({
             'li button:not(:disabled),li input:not(:disabled),li a:not([aria-disabled="true"])'
           );
           firstElement && (firstElement as HTMLElement).focus();
-        }, 0);
+        }, 10);
```

Comment:
@tlabaj wdyt about using a (beta) prop for this timeout amount? Wondering just based on how this issue has come up several times if we can rely on a static timeout without the potential to have to slowly increase it if the issue comes up again.

Reply:
Yes.  we can do that as follow up.